.github/workflows/pr_and_main.yml

```diff
@@ -85,10 +85,29 @@ jobs:
 
       - run: tox -e linkcheck
         # Linkcheck can be flaky. Avoid randomly breaking PR builds by running only on `main`
-        if: ${{ contains(matrix.os, 'ubuntu') && github.ref == 'refs/heads/main' }}
+        if: ${{ contains(matrix.variant.os, 'ubuntu') && github.ref == 'refs/heads/main' }}
 
       - uses: actions/upload-artifact@v2
         if: ${{ contains(matrix.variant.os, 'ubuntu') }}
         with:
           name: DocumentationHTML
           path: html/
+
```

Comment:
What is the problem with running it from a separate workflow?

Reply:
No problem really, there would be two options:

- Make it a reusable workflow, which uses an uploaded artifact (`install` folder) from the CI workflow.
- Stand-alone, building scipp from scratch.